src/sagemaker/jumpstart/utils.py

```diff
@@ -232,6 +232,19 @@ def add_single_jumpstart_tag(
     return curr_tags
 
 
+def get_jumpstart_base_name_if_jumpstart_model(
+    *uris,
```

Comment:
could you refine the typing please, ie would one of these work `*uris: List[Optional[str]]` or `*uris: Sequence[Optional[str]]` ?